src/internal_modules/roc_address/protocol_map.h

```diff
@@ -66,6 +68,12 @@ class ProtocolMap : public core::NonCopyable<> {
     //! Get protocol attributes by scheme name.
     const ProtocolAttrs* find_by_scheme(const char* scheme) const;
 
+    //! Get list of interfaces with at least one protocol
+    ROC_ATTR_NODISCARD bool get_supported_interfaces(core::Array<Interface>&);
+
+    //! Get all supported protocols
+    ROC_ATTR_NODISCARD bool get_supported_protocols(Interface, core::StringList&);
```

Comment:
nitpick: Please add dot at the end of each doxygen comment, like:

```
//! Get list of interfaces with at least one protocol.
```

IIRC doxygen auto-brief mode that we use requires it.